static-analysis/teammates-tslint.yml

```diff
@@ -81,7 +81,10 @@ rules:
   label-position: true
   max-line-length: [
     true,
-    120
+    {
+      "limit": 120,
+      "ignore-pattern": "^import [^,]+ from |^export | implements "
```

Comment:
I think this change is irrelevant to this PR?

Reply:
There are some long imports in this PR, this is to pass the  lint